datastore/datastore.go

```diff
@@ -53,11 +53,20 @@ const DetectProjectID = "*detect-project-id*"
 // the resource being operated on.
 const resourcePrefixHeader = "google-cloud-resource-prefix"
 
+// DefaultDatabaseID is ID of the default database
```

Comment:
A note about how it's empty string by default would be good. Otherwise this looks like a typo.